### PR TITLE
[Fix #66] fix owl2java-transform config and mapping

### LINKED ISSUE #66: Fix jopa-maven-plugin configuration
Fix jopa-maven-plugin configuration so it is able to generate the JAVA model into the correct directories.

### PR DESCRIPTION
**Pull request by grigobog**

### CODE REVIEW

File: bpmn2stamp-converter/pom.xml [hunk 1]
```diff
@@ -201,39 +201,36 @@
...
-<!--                    <mapping-file>${project.basedir}/src/main/resources/jopa/mapping/mapping.map</mapping-file>-->
-<!--&lt;!&ndash;                    <context-name>plan-execution</context-name>&ndash;&gt;-->
-<!--&lt;!&ndash;                    <whole-ontology-as-ics>true</whole-ontology-as-ics>&ndash;&gt;-->
-<!--&lt;!&ndash;                    <vocabulary-only>true</vocabulary-only>&ndash;&gt;-->
-<!--&lt;!&ndash;                    <ignore-failed-imports>true</ignore-failed-imports>&ndash;&gt;-->
-<!--                </configuration>-->
+                    <output-directory>${project.basedir}/src/main/java/cz/cvut/kbss/bpmn2stamp/converter</output-directory>
+                    <mapping-file>${project.basedir}/src/main/resources/jopa/mapping/mapping-owl2java-transform.map</mapping-file>
+                </configuration>
+                <!-- This plugin execution was disabled due to stability of the application -->
```
Comment (blcham) [inline bpmn2stamp-converter/pom.xml:224]:
```suggestion
                <!-- This plugin execution is disabled as the model of JOPA entities is modified (i.e., different from the generated model). Uncomment it to regenerate the model. -->
```

File: bpmn2stamp-converter/pom.xml [hunk 2]
```diff
@@ -201,39 +201,36 @@
...
+                <groupId>cz.cvut.kbss.jopa</groupId>
+                <artifactId>jopa-maven-plugin</artifactId>
+                <version>${cz.cvut.kbss.jopa.version}</version>
+                <configuration>
+                    <!-- Uncomment one of the following sections (only one at a time) to generate the corresponding model -->
+                    <!-- STAMP -->
 <!--                    <package>model.stamp</package>-->
 <!--                    <ontology-iri>http://onto.fel.cvut.cz/ontologies/stamp</ontology-iri>-->
-<!--                    &lt;!&ndash; BBO &ndash;&gt;-->
+                    <!-- BBO -->
```
Comment (blcham) [inline bpmn2stamp-converter/pom.xml:214]: move this to execution

File: bpmn2stamp-converter/pom.xml [hunk 3]
```diff
@@ -201,39 +201,36 @@
...
 
-<!--                    <output-directory>${project.basedir}/src/main/java</output-directory>-->
-<!--                    <mapping-file>${project.basedir}/src/main/resources/jopa/mapping/mapping.map</mapping-file>-->
-<!--&lt;!&ndash;                    <context-name>plan-execution</context-name>&ndash;&gt;-->
-<!--&lt;!&ndash;                    <whole-ontology-as-ics>true</whole-ontology-as-ics>&ndash;&gt;-->
-<!--&lt;!&ndash;                    <vocabulary-only>true</vocabulary-only>&ndash;&gt;-->
-<!--&lt;!&ndash;                    <ignore-failed-imports>true</ignore-failed-imports>&ndash;&gt;-->
-<!--                </configuration>-->
+                    <output-directory>${project.basedir}/src/main/java/cz/cvut/kbss/bpmn2stamp/converter</output-directory>
+                    <mapping-file>${project.basedir}/src/main/resources/jopa/mapping/mapping-owl2java-transform.map</mapping-file>
```
Comment (blcham) [inline bpmn2stamp-converter/pom.xml:222]: please remove file mapping.map if you are not using it.

**Review by blcham — CHANGES_REQUESTED** [review]

File: bpmn2stamp-converter/pom.xml (hunk 3 above)
Comment (grigobog) [reply bpmn2stamp-converter/pom.xml:215]: unused file mapping.map was removed

File: bpmn2stamp-converter/pom.xml (hunk 2 above)
Comment (grigobog) [reply bpmn2stamp-converter/pom.xml]: moved to multiple executions for every model (bbo, stamp, bbo-e)

**Review by blcham — APPROVED** [review]
LGTM, just update README.md and you can merge

File: bpmn2stamp-converter/pom.xml [hunk 4]
```diff
@@ -201,39 +201,55 @@
...
+                        <id>generate-bbo-e</id>
+                        <goals><goal>owl2java-transform</goal></goals>
+                        <phase>none</phase>
+                        <configuration>
+                            <ontology-iri>http://onto.fel.cvut.cz/ontologies/bbo</ontology-iri>
+                            <pPackage>cz.cvut.kbss.bpmn2stamp.converter.model.bbo</pPackage>
+                            <package/>
+                        </configuration>
+                    </execution>
+                    <!-- Generate STAMP model by running jopa:owl2java-transform@generate-stamp maven plugin -->
```
[the remark this answers is not in the document]
Comment (blcham) [reply bpmn2stamp-converter/pom.xml:229]: resolved

**Pull request merged**